compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -1152,11 +1154,13 @@ public void initPredeclaredModules(Map<Name, BPackageSymbol> predeclaredModules,
                                        List<BLangCompilationUnit> compUnits, SymbolEnv env) {
         SymbolEnv prevEnv = this.env;
         this.env = env;
-        for (Name alias : predeclaredModules.keySet()) {
+        Name alias;
+        for (Map.Entry<Name, BPackageSymbol> e : predeclaredModules.entrySet()) {
+            alias = e.getKey();
             int index = 0;
             ScopeEntry entry = this.env.scope.lookup(alias);
             if (entry == NOT_FOUND_ENTRY && !compUnits.isEmpty()) {
-                this.env.scope.define(alias, dupPackageSymbolAndSetCompUnit(predeclaredModules.get(alias),
+                this.env.scope.define(alias, dupPackageSymbolAndSetCompUnit(e.getValue(),
```

Comment:
We can extract `e.getValue()` out to a variable.

Reply:
yes, will do it.

Reply:
Added. bd1ee052c669add9be0a463d609f5286cf4ee229